### DISCUSSION TITLE
Should loadMacros automatically load PGcourse.pl

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork2#1606] [Ideas]
It has become a standard that all OPL problems load PGcourse.pl as the last macro file.  Would it make more sense to have `loadMacros` automatically load PGcourse.pl even if it isn't explicitly listed?

**Comment by drgrice1** [comment]
I don't think that the `loadMacros` method should do this, but it would be nice if it were automatically loaded for all problems.  The reason that I don't think that `loadMacros` should do this is because `loadMacros` can be called multiple times in a problem or in another macro that is loaded.  I know there are protections against macros being loaded multiple times, but those protections are not as robust as they should be (for example the init method is always executed).
